synapse/federation/transport/client.py

```diff
@@ -429,14 +435,29 @@ def query_user_devices(self, destination, user_id, timeout):
         Response:
             {
               "stream_id": "...",
-              "devices": [ { ... } ]
-            }
+              "devices": [ { ... } ],
+              "master_key": {
+                "user_id": "<user_id>",
+                "usage": [...],
+                "keys": {...},
+                "signatures": {
+                  "<user_id>": {...}
+                }
+              },
+              "self_signing_key": {
+                "user_id": "<user_id>",
+                "usage": [...],
+                "keys": {...},
+                "signatures": {
+                  "<user_id>": {...}
+                }
+            } }
```

Comment:
Aren't there too many braces here? again, it's hard to read with multiple closing braces on one line.

Reply:
One brace matches `self_signing_key`, and the other matches the encompassing dict.

The multiple-braces-on-one-line was convention on these methods before I got here, and I kind of like it, but can change to the standard if you like.

Reply:
> One brace matches self_signing_key, and the other matches the encompassing dict.

oh. yes.

but still: I find it hard to read and I think splitting them out would be clearer.